packages/llvm17_build.rb

```diff
@@ -33,12 +31,12 @@ class Llvm17_build < Package
   depends_on 'libxml2' # R
   depends_on 'llvm17_dev' => :build
   depends_on 'ncurses' # R
+  depends_on 'ocaml' => :build
+  depends_on 'py3_pygments' => :build
   depends_on 'py3_pyyaml' => :build
   depends_on 'xzutils' # R
   depends_on 'zlibpkg' # R
   depends_on 'zstd' # R
-  depends_on 'llvm17_dev' # R
```

Comment:
Assuming removing these dependencies was intentional and not an accident (these dependencies looked pretty important, so just checking), then looks good!

Reply:
It's just a reordering. And llvm17_dev pulls in llvm17_lib...